server.py

```diff
@@ -727,6 +732,37 @@ def post(self, region, id):
         except:
             return 'Dao threw an error somewhere', 400
 
+class AddTournamentMatchResource(restful.Resource):
+    def get(self, region, id):
+        pass
+
+    def put(self, region, id):
+        dao = Dao(region, mongo_client=mongo_client)
+        user = get_user_from_request(request, dao)
+
+        args = tournament_details_add_match_parser.parse_args()
+        tournament = dao.get_tournament_by_id(ObjectId(id))
```

Comment:
probably should wrap this in a try/except in case there is an error casting to ObjectId